lib/middleware/conversations/create_conversation.dart

```diff
@@ -1,6 +1,6 @@
 import 'package:crowdleague/actions/conversations/create_conversation.dart';
```

Comment:
Just changing from old navigation strategy to using new navigator

Reply:
![navigator](https://pmcvariety.files.wordpress.com/2017/09/flight-of-the-navigator.jpg?w=1000)

Reply:
hahahaha I loved this movie!!!